2024-10-13 19:46:31.040266: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2024-10-13 19:46:31.044830: I external/local_xla/xla/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2024-10-13 19:46:31.059090: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:485] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-10-13 19:46:31.082553: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:8454] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-10-13 19:46:31.089546: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1452] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-10-13 19:46:31.106950: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-10-13 19:46:32.300383: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/user/gpr_ai_project/cnn_model.py", line 115, in <module>
    cnn_gpr.train(train_dir='data/train', validation_dir='data/validation', epochs=5, batch_size=2)
  File "/home/user/gpr_ai_project/cnn_model.py", line 55, in train
    train_generator = train_datagen.flow_from_directory(
  File "/usr/local/lib/python3.10/dist-packages/keras/src/legacy/preprocessing/image.py", line 1138, in flow_from_directory
    return DirectoryIterator(
  File "/usr/local/lib/python3.10/dist-packages/keras/src/legacy/preprocessing/image.py", line 453, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: 'data/train'
